timemachine/md/minimizer.py

```diff
@@ -551,3 +551,17 @@ def val_and_grad_fn_bfgs(x_local_flattened):
         warnings.warn(f"WARNING: Energy did not decrease: u_0: {u_0:.3f}, u_f: {U_final:.3f}", MinimizationWarning)
 
     return x_final
+
+
+def replace_conformer_with_minimized(mol: Chem.rdchem.Mol, ff: Forcefield):
+    top = topology.BaseTopology(mol, ff)
+    system = top.setup_end_state()
+    val_and_grad_fn = get_val_and_grad_fn(system.get_U_fns(), 1e9 * np.eye(3))
+    xs = get_romol_conf(mol)
+    all_idxs = np.arange(mol.GetNumAtoms())
+    xs_opt = local_minimize(xs, val_and_grad_fn, all_idxs, verbose=False)
+    conf = Chem.Conformer(mol.GetNumAtoms())
```

Comment:
Would be good to use `set_romol_conf` here

Reply:
Thanks, that's much cleaner. Fixed in https://github.com/proteneer/timemachine/pull/1229/commits/2037d231e40767a8f34f21def108c2d4b54ce91e.